### ISSUE TITLE
Landmarks problem

### ISSUE THREAD

**Issue by baba-yaga** [BoomStarcuc/3DSfMFaceReconstruction#4]
Hello,
I have tried the landmark detection on my images (just replaced the references in the code Facial_landmak.py and they do not appear to detect any significant features of the face and they do not match between the images.
The images are png squared 384x384 as yours used in test_landmark folder. What am I doing wrong?

**Comment by wahiahia** [comment]
Is your problem solved? Or is there a better solution for matching two face images?

**Comment by baba-yaga** [comment]
No. I believe this project is abandoned so I am looking for another solution

**Comment by wahiahia** [comment]
do you have any good advice
